lib_eio/flow.ml

```diff
@@ -161,6 +161,12 @@ let buffer_sink =
   let ops = Pi.sink (module Buffer_sink) in
   fun b -> Resource.T (b, ops)
 
+let load (t : _ source) =
+  let b = Buffer.create 4096 in
```

Comment:
Would it make sense to have the buffer size be configurable through an optional parameter? `?(size = 4096)`?

Reply:
Possibly. I debated whether to do it or not. Ultimately I went with no because `Buffer_sink.copy` just 20 lines above also doesn't. Plus I figured that any situation where the buffer size would have an impact would not use a function that loads everything into a single large string; it would process the file in small chunks instead. It made sense to me to keep this "beginner function" as simple as possible.